### PR TITLE
Update to Vert.x 3.9.1

### PR DESCRIPTION
**Pull request by cescoffier**
Update the following versions:

Vert.x -> 3.9.1
SmallRye Reactive Messaging -> 2.0.4
SmallRye Reactive Streams Operators and Reactive Converters -> 1.0.13
SmallRye Reactive Utils and Vert.x Mutiny Client -> 0.0.16

### CODE REVIEW

File: extensions/grpc/runtime/pom.xml [hunk 1]
```diff
@@ -9,10 +9,6 @@
         <version>999-SNAPSHOT</version>
     </parent>
 
-    <properties>
-        <vertx.version>3.9.0</vertx.version>
-    </properties>
```
Comment (gsmet) [inline extensions/grpc/runtime/pom.xml:14]: Good catch!

**Review by gsmet — APPROVED** [review]
Looks good to me!

**Comment by gsmet** [comment]
@cescoffier you can promote it!

**Comment by gsmet** [comment]
Rebased and promoted.

**Comment by stuartwdouglas** [comment]
That failure is very odd, I have no idea why you would get a 403 in this test, it should not be possible.

**Comment by stuartwdouglas** [comment]
I can't explain that failure, it is using Undertow to run the test, even though Undertow is not a dependency. One of the tests in that module uses Undertow as a 'forced dependency', but this should not affect the failed test. Running it locally it uses RESTEasy standalone as expected.

**Comment by stuartwdouglas** [comment]
Failures should be fixed by https://github.com/quarkusio/quarkus/pull/9525

**Comment by geoand** [comment]
PR rebased onto master and force pushed

**Pull request merged**